app/views/products/_product.html.erb

```diff
@@ -17,12 +20,27 @@
   </td>
   <td data-order="<%= modified_time.to_i %>"><%= modified_time.strftime("%m/%d/%y") %></td>
   <td>
-    <%= link_to 'Shell', OodAppkit.shell.url(path: product.router.path).to_s, target: '_blank', class: 'btn btn-default btn-block' %>
+    <%= link_to 'Shell', OodAppkit.shell.url(host: ENV['OOD_DEV_SSH_HOST'] || 'default', path: product.router.path).to_s, target: '_blank', class: 'btn btn-default btn-block' %>
     <%= link_to 'Files', OodAppkit.files.url(path: product.router.path).to_s, target: '_blank', class: 'btn btn-default btn-block' %>
   </td>
   <td>
     <%= link_to 'Details', product_path(product.name, type: @type), class: 'btn btn-info btn-block' %>
     <%= link_to 'Edit', edit_product_path(product.name, type: @type), class: 'btn btn-default btn-block' %>
   </td>
-  <td><%= link_to 'Delete', product_path(product.name, type: @type), method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger pull-right' %></td>
+  <td>
```

Comment:
I have a feeling this string/button is more likely to change than the others:

>Unexpected bad things will happen if you don\'t read this!</div><p>This will move the product to the trash. Any users currently running this app will experience breaking behavior. Any past or present users of this app will be unable to access their data for this app. Be sure you know what you are doing!

Or the title:

> Are you ABSOLUTELY sure?

Could be wrong. I think breaking convention here is okay with a helper. My guess is it would be something like 

```erb
<%= delete_product_btn(@product, @type) %>
```

But if you feel strongly against it we can keep it as it is.